widgetIDESupport/index.js

```diff
@@ -493,6 +515,7 @@ if (TW.IDE && (typeof TW.IDE.Widget == 'function')) {
                 TWComposerWidget.prototype.widgetProperties = prototype.widgetProperties;
                 TWComposerWidget.prototype.widgetEvents = prototype.widgetEvents;
                 TWComposerWidget.prototype.widgetServices = prototype.widgetServices;
+                TWComposerWidget.prototype.afterLoad = property.afterLoad;
```

Comment:
There seems to be a typo here

Reply:
Good catch. I've pushed a fix and retested, however, the problem still persists and i'm seeing the property name, not the display name.